containers/Connector/config.ts

```diff
@@ -20,6 +20,12 @@ export const initOnboard = (network: Network, subscriptions: Subscriptions) => {
 		walletSelect: {
 			wallets: [
 				{ walletName: 'metamask', preferred: true },
+				{
+					walletName: 'lattice',
+					appName: 'Synthetix',
+					rpcUrl: infuraRpc,
+					preferred: true,
```

Comment:
@MidnightOnMars can you please remove `preferred: true`

Reply:
Updated to remove preferred status.